zulip/integrations/google/doc.md

```diff
@@ -1,72 +1,172 @@
-Get Google Calendar reminders in Zulip! This is a great way to see
-your reminders directly in your Zulip feed.
+# Zulip Google Calendar integration
+
+Get Zulip notifications for your Google Calendar events!
+
+### Create Zulip bot for Google Calendar notifications
+
+{start_tabs}
+
+1.  {!create-a-generic-bot.md!}
 
 1.  {!download-python-bindings.md!}
 
-    This bot should be set up on a trusted machine, because your API
-    key is visible to local users through the command line or config
-    file.
+1.  [Download your bot's `zuliprc` file][download-zuliprc], and save it as
+    `.zuliprc` in your `~/` directory.
```

Comment:
Also, just confirming that this can be done on any machine. It doesn't have to be the one running the Zulip server, correct? I think this is probably clear for a server admin, but if a general user or an organization admin was looking at setting up this integration, we want it to be clear for them.